ci/README.md

```diff
@@ -6,6 +6,11 @@ Since DMD runs on many platforms and lots of D infrastructure depends on it, the
 ## Troubleshooting failed checks
 When your PR fails checks, click on the "Details" link next to it to see what the error is.
 
+Most log files are the output of `make`, which prints the commands it invokes followed by the output of that command.
+Execution stops after a failed command, so errors can usually be found near the end of the log.
+Because of multi-threaded builds, they don't need to be at the very end however.
+You can Ctrl+F search for a triple asterisk `***` to easily find failed commands.
```

Comment:
Regarding Ctrl+F:

![image](https://user-images.githubusercontent.com/1889215/178071327-d19f7acd-0161-403e-93b0-704ac8a4288a.png)

@WalterBright: That's what I meant for Azure CI (but also applies to GitHub Actions) - don't use Ctrl+F for the browser search (see toolbar at the bottom - no previous hits for `testabi`), but first focus the 'log' panel and then press Ctrl+F for the log-internal search, which is the one that actually works.

I usually grep for ` failed.` (case-sensitive) for the DMD tests (not using make unlike druntime+Phobos).